zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/BabbagePdfService.java

```diff
@@ -48,19 +49,20 @@ public InputStream generatePdf(String uri) throws IOException {
 
         logInfo("Reading PDF").addParameter("src", src).log();
 
-        // if the url is absolute, go get it using HTTP client.
-        HttpClient client = HttpClientBuilder.create().build();
-        HttpGet httpGet = new HttpGet(src);
-        httpGet.addHeader("Cookie", "access_token=" + session.id);
-        httpGet.addHeader("Cookie", "collection=" + collection.description.id);
-        HttpResponse response = client.execute(httpGet);
+        try (CloseableHttpClient client = HttpClientBuilder.create().build()) {
+            HttpGet httpGet = new HttpGet(src);
+            httpGet.addHeader("Cookie", "access_token=" + session.id);
+            httpGet.addHeader("Cookie", "collection=" + collection.description.id);
 
-        if (response.getStatusLine().getStatusCode() != 200) {
-            throw new IOException("Failed to generate PDF for URI " + uri +
-                    ". Response: " + response.getStatusLine().getStatusCode() +
-                    " " + response.toString());
-        }
+            try (CloseableHttpResponse response = client.execute(httpGet)) {
```

Comment:
I did check this locally before pushing this change and I didn't get this issue. I will check again anyway.